packages/backend/src/database/migrations/20240108152115_group_attributes.ts

```diff
@@ -0,0 +1,30 @@
+import { Knex } from 'knex';
+
+const userAttributesTableName = 'user_attributes';
+const groupUserAttributes = 'group_user_attributes';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(groupUserAttributes))) {
+        await knex.schema.createTable(groupUserAttributes, (table) => {
+            table.uuid('group_uuid').notNullable();
```

Comment:
We could combine this line with line 11 so all the instructions for the same column are chained. 
And do the same for the lines for `user_attribute_uuid`.

Reply:
Knex doesn't seem to like them chained when its a foreign key? 

This give an error, for example

`table
    .uuid('group_uuid')
    .foreign('group_uuid')
    .references('group_uuid')
    .inTable('groups')
     .onDelete('CASCADE');`

Reply:
If you put it on the column you don't need `.foreign`

```
table.uuid('group_uuid')
.notNullable()
.references('group_uuid')
.inTable('groups')
.onDelete('CASCADE')
```